src/components/App/App.js

```diff
@@ -8,15 +8,18 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
+      movies: [],
+      error: ''
     }
   }
 
-  // componentDidMount() {
-  // put the fetch here
-  // }
+  componentDidMount() {
+    fetch(' https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
```

Comment:
This looks great, the only suggestion I'd have is to put a conditional in the first `.then()` to handle errors like 422 that don't get sent to the `.catch()` automatically. It could look like `if (!res.ok) { throw new Error('oopsie daisy') }` or something. Probably not super necessary for the project as is but could be a good thing to implement for thinking long-term (if we ever wanted to incorporate POST requests & stuff).